app/assets/javascripts/central.js

```diff
@@ -61,5 +62,9 @@ var Central = module.exports = {
 
       window.projectView = view;
     });
+
+    setTimeout(function() {
```

Comment:
Since you don't use `this` inside `firstTimeTour.start`, you could do just: `setTimeout(firstTimeTour.start, 200)` (notice we're not _calling_ start, we're passing the function itself to `setTimeout`)